nested-spire/scripts/gencerts.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+func main() {
+	if len(os.Args) < 3 {
+		fmt.Fprintln(os.Stderr, "usage: gencerts SERVERDIR AGENTDIR [AGENTDIR...]")
+		os.Exit(1)
+	}
+
+	notAfter := time.Now().Add(time.Minute * 10)
```

Comment:
it will expire too soon, maybe you can increment it to 2hs? or something like that? just to allow them play with it before it expires